client/me/notification-settings/index.jsx

```diff
@@ -74,3 +75,16 @@ export default React.createClass( {
 	}
 } );
 
+export default connect(
+	() => {
+		return {}
+	},
+	( dispatch ) => { return {
```

Comment:
This is the place I would like to have some helper, because implementing this action creators in every place that uses notices would be... verbose.